packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -137,22 +136,20 @@ class ReactFabricHostComponent implements NativeMethods {
   }
 
   measure(callback: MeasureOnSuccessCallback) {
-    const {stateNode} = this._internalInstanceHandle;
-    if (stateNode != null) {
-      fabricMeasure(
-        stateNode.node,
-        mountSafeCallback_NOT_REALLY_SAFE(this, callback),
```

Comment:
`mountSafeCallback_NOT_REALLY_SAFE` was always a no-op.

Reply:
what do you mean it was a noop? It seems to check mounted flag.

Reply:
The mounted flag is added by `createReactClass` to the class instance, which doesn't match instances of `ReactFabricHostComponent`. That means this code doesn't handle anything.